tests/integration_tests/test_sampler.py

```diff
@@ -61,6 +61,7 @@ def objective0(trial):
         p2 = trial.suggest_int('p2', 0, 10)
         p3 = trial.suggest_discrete_uniform('p3', 0, 9, 3)
         p4 = trial.suggest_categorical('p4', ['10', '20', '30'])
+        assert isinstance(p4, str)
```

Comment:
These assertions (there are several in this PR) are required to pass mypy and are not just there for unit test purposes.